### ISSUE TITLE
[Creator Starter Kit] BUG: Fluent UI details list ColCellType : Image

### ISSUE THREAD

**Issue by kaalkit** [microsoft/powercat-creator-kit#350]
**Describe the bug**
Fluent UI details list code component is utilized in the canvas app, unfortunately I am unable to add an alt text to a **ColCellType : Image. This is impacting our ITAS testing, the jaw software reads the image as unlabeled graphic.**

**Expected behavior**
Need a property to where we can specify the value for attribute of image type's "alt" text.
